### ISSUE TITLE
Proposals for a logo

### ISSUE THREAD

**Issue by messismore** [OttoWinter/esphomeyaml#53]
Hi!

Inspired by this [Twitter thread](https://twitter.com/TheHookUp1/status/1005860946278051840) I have done some rough sketches for what I think would make a nice logo, or, rather a starting point. They are pretty straightforward in their metaphor (a stylised ESP8266 as mounted on the NodeMCU in varying degrees of abstraction + a house) but I think the simplicity works rather well.
![nodemcu_amica_bot_02](https://user-images.githubusercontent.com/5689189/41376414-77de9f52-6f59-11e8-8fe5-9f9b9ce7b7b3.png)

I really liked @OttoWinter's idea to work with the chip's antenna, as it is quite a distinct feature, however I am also drawn to no.1 visual simplicity. Maybe someone will be inspired to a eureka moment by one of these.

Note that I kept the name esphome*lib* for now, as @OttoWinter is still thinking about the name.

![esphomelib logo proposal-01](https://user-images.githubusercontent.com/5689189/41376269-0b41dbf2-6f59-11e8-8603-7f92a9ac39be.png)
[esphomelib logo proposal.pdf](https://github.com/OttoWinter/esphomeyaml/files/2099753/esphomelib.logo.proposal.pdf)

Cheers

**Comment by carlosmgr** [comment]
I Like the 5, it's simple

**Comment by brandond** [comment]
I like 3 shows the recognizable antenna, plus I like the home being 'in' the chip.

**Comment by Landrash** [comment]
I give a vote for 5 and 3.

I suggest @OttoWinter adopt the one he likes but having svg/png files available in the documentation repository would be rather awesome.

@messismore Have you considered nr 3 with a hollow house instead?

**Comment by messismore** [comment]
>  I would be happy to incorporate the design into the website and project if you're okay with it

Absolutely! Should you decide to adopt one of them, I'd go over it some (harmonise it with the font's dimensions) and issue a pull request with a Svg and Png.

@Landrash I did, but I think the solid house provides some nice visual weight next to the wordmark:
<img width="835" alt="esphomelib 3b" src="https://user-images.githubusercontent.com/5689189/41476564-3793e834-70c2-11e8-8f37-410aa5ac9d6c.png">

**Comment by jackjohnsonuk** [comment]
Number 5 👍

**Comment by Landrash** [comment]
@messismore I agree with you on that it looks beter with the filled house.

**Comment by mefizto** [comment]
3

**Comment by zmrow** [comment]
My vote is #3

**Comment by cgtobi** [comment]
My votes for 5 and 3, in that order. :) Great logos. They'd be recognisable on their own without the text quite nicely.

**Comment by justinmartin** [comment]
Number 3 is tops for me. Followed closely by 5.

**Comment by OttoWinter** [comment]
@messismore Sorry for the waiting. I've had to concentrate on other topics for a while (work, ioi, university, ...). I would be really happy to finally see esphomelib with a logo and I've spent the past time contemplating a lot about the marvelous logos 5 & 3 and in this thread they both seem to share about the equal amount of fans :)

For the next release I would really like to have the new logo ready. And.... I have decided that I want to go for number 3 :) When you have time, could you do the final cleanups you wanted to do and create a PR with an image file? Anyway, if you don't have time right now (which is 💯 ok), I mean I can also just use a cropped picture from above and maybe update it later.

Thank you again very much. Your work is excellent and very much appreciated!
